server/controllers/class/recommended.ts

```diff
@@ -0,0 +1,9 @@
+import { Request, Response } from 'express';
+import recommendedQueries from '../../queries';
+
+const recommended = async (req: Request, res: Response) => {
+  const { classId } = req.params;
+  res.json(await recommendedQueries(Number(classId)));
```

Comment:
Add your code inside try and catch so you can catch any error in `recommendedQueries`.
Another thing is to send the response as this pattern `json({ data })`